hive-x-bigquery-connector/pom.xml

```diff
@@ -0,0 +1,134 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This module is inherited from by the hive3 module, and so by the hive2 module in a future PR. Meant to gather configuration that is common between all the connector versions.